docs/.vitepress/vitepress/components/vp-demo.vue

```diff
@@ -126,25 +106,6 @@ const copyCode = async () => {
             </a>
           </ElIcon>
         </ElTooltip>
-        <ElTooltip
-          :content="locale['copy-code']"
-          :show-arrow="false"
-          :trigger="['hover', 'focus']"
-          :trigger-keys="[]"
-        >
-          <ElIcon
-            :size="16"
-            :aria-label="locale['copy-code']"
-            class="op-btn"
-            tabindex="0"
-            role="button"
-            @click="copyCode"
-            @keydown.prevent.enter="copyCode"
-            @keydown.prevent.space="copyCode"
-          >
-            <i-ri-file-copy-line />
-          </ElIcon>
-        </ElTooltip>
```

Comment:
IMO, It might be better to keep this copy button so that users don't have to expand the code first before copying it.

Reply:
I think there's no harm in keeping them both(ep copy & vitepress copy in code), partly to be compatible with past user habits.

Reply:
Well, I will revert the code tomorrow.